### PR TITLE
fix(ui): missing components on themes

### PR DESCRIPTION
**Pull request by vimercati-samir**
Screenshot 2022-01-14 at 11 56 09
Screenshot 2022-01-14 at 11 56 25
Screenshot 2022-01-14 at 11 56 52
Screenshot 2022-01-14 at 11 57 11
Screenshot 2022-01-14 at 11 57 28
Screenshot 2022-01-14 at 11 57 58

### CODE REVIEW

File: assets/styles/themes/moonless_night.less [hunk 1]
```diff
@@ -361,4 +366,22 @@
   .new-message-alert {
     background: var(--flair-color) !important;
```
Comment (iltumio) [inline assets/styles/themes/moonless_night.less:367]: is there a way to avoid `!important`?

File: assets/styles/themes/moonless_night.less [hunk 2]
```diff
@@ -361,4 +366,22 @@
...
   }
+
+  .tag {
+    .tag-number {
+      background: var(--flair-color) !important;
```
Comment (iltumio) [inline assets/styles/themes/moonless_night.less:372]: same as above

File: assets/styles/themes/moonless_night.less [hunk 3]
```diff
@@ -151,6 +151,11 @@
   .multiselect__tags {
     background: var(--flair-color) !important;
```
Comment (iltumio) [inline assets/styles/themes/moonless_night.less:152]: same as below

**Review by iltumio — DISMISSED** [review]

File: assets/styles/themes/moonless_night.less (hunk 3 above)
Comment (vimercati-samir) [reply assets/styles/themes/moonless_night.less]: resolved in new commit

File: assets/styles/themes/moonless_night.less (hunk 2 above)
Comment (vimercati-samir) [reply assets/styles/themes/moonless_night.less]: resolved in new commit

File: assets/styles/themes/moonless_night.less (hunk 1 above)
Comment (vimercati-samir) [reply assets/styles/themes/moonless_night.less:367]: new-message-alert style has not been affected by AP-384, nonetheless, should I investigate?

**Comment by WanderingHogan** [comment]
I believe this is good to go in, and the !important from before is something we should come back to in a larger ticket where we review all of the style rules/overrides

**Review by WanderingHogan — APPROVED** [review]

**Comment by WanderingHogan** [comment]
Sorry - not ready to go in, but ready for QA

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**